docs/guide/design/icon-design-guide.md

```diff
@@ -6,56 +6,109 @@ title: Icon Design Guide
+### 1. Icons must be designed on a **24 by 24 pixels** canvas.
 
 ![24px-24px](../../images/24px-24px.svg?raw=true "24px-24px")
 
-### 2. Icons must have at least 1 pixel padding within the canvas.
+### 2. Icons must have at least **1 pixel padding** within the canvas.
 
 ![1px-padding](../../images/1px-padding.svg?raw=true "1px-padding")
 
-### 3. Icons must have a stroke width of 2 pixels.
+### 3. Icons must have a **stroke width of 2 pixels**.
 
 ![2px-stroke](../../images/2px-stroke.svg?raw=true "2px-stroke")
 
-### 4. Icons must use round joins.
+### 4. Icons must use **round joins**.
 
 ![round-joints](../../images/round-joints.svg?raw=true "round-joints")
 
-### 5. Icons must use round caps.
+### 5. Icons must use **round caps**.
 
 ![round-caps](../../images/round-caps.svg?raw=true "round-caps")
 
-### 6. Icons must use centered strokes.
+### 6. Icons must use **centered strokes**.
 
 ![centered-strokes](../../images/centered-strokes.svg?raw=true "centered-strokes")
 
-### 7. Shapes (such as squares) in icons must have border radius of 2 pixels.
+### 7. Shapes (such as rectangles) must have a **border radius of**
+
+#### C. **2 pixels** if they are at least 8 pixels in size
 
 ![2px-border-radius](../../images/2px-border-radius.svg?raw=true "2px-border-radius")
 
-### 8. Distinct elements must have 2 pixels of spacing between each other.
+#### B. **1 pixel** if they are smaller than 8 pixels in size
+
+![1px-border-radius](../../images/1px-border-radius.svg?raw=true "1px-border-radius")
+
+### 8. Distinct elements must have **2 pixels of spacing between each other**
 
 ![2px-element-spacing](../../images/2px-element-spacing.svg?raw=true "2px-element-spacing")
 
-## Code Conventions
+### 9. Icons should have a similar optical volume to `circle` and `square`.
 
-Before an icon is added to the library, we like to have readable and optimized svg code.
+![optical-volume-ideal](../../images/optical-volume-ideal.svg?raw=true "optical-volume-ideal")
 
-Never use [`<use>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use). While it may sometimes seem like a good way to optimize file size, there's no way to ensure that the referenced element IDs will be unique once the SVGs are embedded in HTML documents.
+![optical-volume-low](../../images/optical-volume-low.svg?raw=true "optical-volume-low")
+
+![optical-volume-high](../../images/optical-volume-high.svg?raw=true "optical-volume-high")
+
+**Tip:** place your icon next to circle or square and blur them both; your icon should not feel much darker than the base shape.
+
+### 10. Icons should be visually centered by their center of gravity.
+
+![visually-centered](../../images/visually-centered.svg?raw=true "visually-centered")
+
+![visually-centered-bad](../../images/visually-centered-bad.svg?raw=true "visually-centered-bad")
+
+**Tip:** place your icon next to and below or above square or circle and check if it feels off centre. Symmetrical icons should always be aligned to the center.
```

Comment:
Maybe slightly different wording:
```diff
- **Tip:** place your icon next to and below or above square or circle and check if it feels off centre. Symmetrical icons should always be aligned to the center.
+ **Tip:** place your icon below, above or next to the square or circle and check if it feels off centre. Symmetrical icons should always be aligned to the center.
```

Reply:
```suggestion
**Tip:** place your icon both next to and below or above the square or circle icon and check if it feels off center. Symmetrical icons should always be aligned to the center.
```

Reply:
I'm not a fan of putting the next to first, I would just put it last since it's kind of like a catch-all. I would also avoid `/` if possible and just write or instead. My suggestion:
```suggestion
**Tip:** place your icon above, below or next to the square or circle and check if it feels off centre. Symmetrical icons should always be aligned to the center.
```

Reply:
These do not mean the same thing though, icons should be checked to be horizontally _and_ vertically centered.

Reply:
Then I would suggest:

 place your icon above or below and next to the square or circle and check if it feels off centre.